assets/onboarding/wp-colors.scss

```diff
@@ -46,19 +47,23 @@ body.sensei-color {
 			border-color: $button;
 		}
 
-		&.is-secondary:active:not(:disabled), &.is-tertiary:active:not(:disabled) {
+		&.is-secondary:active:not(:disabled), &.is-tertiary:active:not(:disabled), &.is-secondary:focus:not(:disabled) {
 			color: $button-shade-20;
 			box-shadow: none;
+			border-color: $button-shade-20;
 		}
 
 		&.is-secondary:hover:not(:disabled), &.is-tertiary:hover:not(:disabled) {
 			color: $button-shade-10;
-			box-shadow: inset 0 0 0 1px $button-shade-10;
+			box-shadow: none;
+			border-color: $button-shade-10;
 		}
 
 		&.is-secondary {
-			box-shadow: inset 0 0 0 1px $button;
+			box-shadow: none;
 			color: $button;
+			background-color: transparent;
+			border-color: $button;
```

Comment:
Let's use `isTertiary` for the `Continue` button on the feedback page, and keep the secondary style with the outline, and generally all components looking & behaving like [this](
https://wordpress.github.io/gutenberg/?path=/docs/components-button--default) - we mostly just want to apply our custom color here.
Looks like the `is-tertiary` class needs a bit more styling, the focus outline is still blue on that one.

Reply:
Hi @yscik !

I'm not sure if I understood your point. The `Continue` button you mean is about [this](https://cloudup.com/i7-9uwU3dYu)? If yes, I think it's a secondary because it has the outline, right? I'm using the `isTertiary` in the [confirmation modal](https://senseip2.files.wordpress.com/2020/02/3.-features-modal-1.jpg?w=1024) because it has no border. And it looks good for me (even with the focus that just change the border color to a gray - unless we want another color here).

Reply:
That's fine too on the `Continue` button, I thought you wanted a button there without outline. 

The issue is, the CSS change above removes the outline from the secondary buttons with `box-shadow: none`. Looks like this for me:

![image](https://user-images.githubusercontent.com/176949/82229358-9ffc2700-992a-11ea-9111-bbbc765d68f9.png)

And the blue border on tertiary buttons when focused:
![image](https://user-images.githubusercontent.com/176949/82229413-b1453380-992a-11ea-9293-938e12c2ad14.png)

Reply:
Hum. Weird! For me it's correct. What's the browser that you're using? Did you check if the CSS was correctly built?

Reply:
The change above removes the outline, so what I'm seeing is how it's supposed to look. Build runs and it's the same in Firefox Mac/Linux & Chromium.
Tried out reverting 7f79ebf789bb0cc580ebc03d54b2e3bca88c30b5 and that puts the outline back, so it's definitely the code :)

Reply:
I updated the styles here: https://github.com/Automattic/sensei/pull/3121/commits/dff5dcc52729a08bd7e2f5d84e1e57393bddc15d

I also created the card to build the Gutenberg CSS instead of importing it. https://github.com/Automattic/sensei/issues/3173